test/_functions/python/kafka-explicit-ack/explicitacker.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+    This function simulates usage of the explicit ack feature for kafka triggered functions.
+    When it receives events from the kafka cluster, it writes them to a file and sends a 'no-ack' response.
+    When the function is triggered by a http request, depending on the request it can:
+    1. read the file and return the number of events queued for processing.
+    2. read the file and return the last committed offset.
+    3. start processing the events in the file - ack each of the events explicitly.
+"""
+
+import json
+import os
+import nuclio_sdk
+
+events_file_path = '/tmp/events.json'
+last_committed_offset_file_path = '/tmp/offset.txt'
+
+
+def init_context(context):
+    context.logger.info("Initializing context")
+
+    # create file - first line is queue size, second line is last commit offset
+    if not os.path.exists(events_file_path):
+        with open(events_file_path, "w"):
+            pass
+
+    if not os.path.exists(last_committed_offset_file_path):
+        with open(last_committed_offset_file_path, "w"):
```

Comment:
decide if you are using `"` or `'`. since most (if not, all) of python functions here uses `'`, then it has been decided ;)